### PR TITLE
Allow configuring sidecars on the operator level in the Helm chart

### LINKED ISSUE #1493: Operator Helm chart should support sidecar configuration
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** `registry.opensource.zalan.do/acid/postgres-operator:v1.6.2`
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** Bare Metal K8s
- **Are you running Postgres Operator in production?** [yes | no] yes
- **Type of issue?** [Bug report, question, feature request, etc.] feature request

The operator allows configuring sidecars globally so the Helm chart should support it. Will submit a PR for this.

### PR DESCRIPTION
**Pull request by stephan2012**
Closes #1493

### CODE REVIEW

File: charts/postgres-operator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgres-operator
-version: 1.6.2
+version: 1.6.3-2
```
Comment (FxKu) [inline charts/postgres-operator/Chart.yaml:3]:
```suggestion
version: 1.6.2
```

File: charts/postgres-operator/templates/configmap.yaml [hunk 2]
```diff
@@ -25,4 +25,6 @@ data:
 {{ toYaml .Values.configLoggingRestApi | indent 2 }}
 {{ toYaml .Values.configTeamsApi | indent 2 }}
 {{ toYaml .Values.configConnectionPooler | indent 2 }}
+  sidecars:
+    {{ toYaml .Values.configSidecars | indent 4 }}
```
Comment (FxKu) [inline charts/postgres-operator/templates/configmap.yaml:29]: have you tested specifying sidecars in a configmap? I doubt it supports nested objects. I think one can use it only with the config CRD. But haven't tested myself yet.

Comment (stephan2012) [reply charts/postgres-operator/templates/configmap.yaml:29]: No, I have not. I am just using the operatorConfig.

**Comment by stephan2012** [comment]
> I suppose this is already supported by including helm values like this?

Indeed, it is. Sorry for the noise, closing this PR.

**Pull request closed without merging**